gateway/src/test/kotlin/gateway/DefaultGatewayTest.kt

```diff
@@ -31,8 +33,8 @@ class DefaultGatewayTest {
         val gateway = DefaultGateway {
             client = HttpClient(CIO) {
                 install(WebSockets)
-                install(JsonFeature) {
-                    serializer = KotlinxSerializer(Json)
+                install(ContentNegotiation) {
+                    json()
```

Comment:
and here?

Reply:
It just passed in `Json.Companion`, which also was the default back then